azplugins/flow.py

```diff
@@ -757,3 +776,15 @@ def velocity(self):
             return np.divide(self._velocity, self._counts, out=np.zeros(self.bins), where=self._counts > 0)/self.samples
```

Comment:
Noting here: this line is incorrect. `self._counts` is already taking care of the total number of samples, so this is reducing the velocity by one factor too many.